WordPress/src/main/java/org/wordpress/android/ui/menus/views/MenuItemAdapter.java

```diff
@@ -83,12 +84,12 @@
 
     @Override
     public int getItemViewType(int position) {
-        if (mFlattenedMenuItems != null){
-            MenuItemModel item = mFlattenedMenuItems.get(position);
-            if (item != null) {
-                if (item instanceof AddItem) {
-                    return VIEW_TYPE_ADDER;
-                }
+        if (mFlattenedMenuItems.size() == 0) return VIEW_TYPE_ADDER;
```

Comment:
I would still check if `mFlattenedMenuItems` is != null here, to avoid potential NPE